tsconfig.json

```diff
@@ -1,24 +1,20 @@
 {
+  "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "allowJs": true,
-    "target": "es6",
-    "moduleResolution": "node",
-    "outDir": "dist/es6",
-    "rootDir": "./",
+    "declaration": false,
```

Comment:
nit: Just adding a note that in the final state after the refactoring, this should be removed. Most tools that read `tsconfig.json` are okay with a trailing `// comment` style comment on the same line. It would be worth added a TODO here.

Reply:
Added. I wonder though if this will ever go away since we will likely keep the generated client as is.